test/config.test.js

```diff
@@ -1,6 +1,48 @@
+const faker = require('faker');
+
 const config = require('../lib/config');
+const { getTestAccessToken } = require('../test/test-helpers/auth-helpers');
+
+const validConfig = {
+  apiServerUrl: faker.internet.url(),
+  apiVersion: faker.random.arrayElement(['v3', 'v4', 'v5']),
+  piiServerUrl: faker.internet.url(),
+  clientId: getTestAccessToken(),
+  clientSecret: getTestAccessToken(),
+};
 
 describe('config', () => {
+  describe('init()', () => {
```

Comment:
Nit: A convention when referencing a module or class method is to preface its name with a `#`:
```suggestion
  describe('#init', () => {
```
We're doing the same elsewhere in the library (see https://github.com/rewardops/rewardops-sdk-node/pull/46/files#diff-61b19a774848afe700b799296ff39a82R16), so I think we should continue that for consistency.